exalearn-ui/src/app/Enums/enums.ts

```diff
@@ -0,0 +1,8 @@
+export enum EnglishLevels {
+	A1 = 'Beginner(A1)',
+	A2 = 'Elementary(A2)',
+	B1 = 'Intermediate(B1)',
+	B2 = 'Upper-Intermediate(A2)',
+	C1 = 'Advanced(C1)',
+	C2 = 'Proficiency(C2)'
+}
```

Comment:
Just a suggestion. Maybe space before brackets?
If it looks good on the page, than it's OK.